src/lib/caching/structures/guilds/GuildMember.ts

```diff
@@ -94,6 +94,19 @@ export class GuildMember extends Structure {
 		return this.nick ?? this.user?.username ?? null;
 	}
 
+	/**
+	 * The displayed colour of the member.
+	 * @since 0.0.4
+	 */
+	public get displayColor(): number {
+		let highestRole = null;
+		for (const role of this.roles.values()) {
+			if (!highestRole || role.position < highestRole.position) continue;
```

Comment:
Am I misreading this or is this totally bugged? Above it is defined as `null` then in every iteration this is just going to simply skip the loop. It's `null` so `if (!null )` will always pass and continue to next loop meaning it will never really get the color

Reply:
Second bug on this is that it does not take into account when the role positions are equal. Yes that is possible

Reply:
May I ask when can role positions be equal?

Reply:
@Rexogamer https://github.com/discord/discord-api-docs/issues/1778